filters/auth/doc.go

```diff
@@ -1,18 +1,162 @@
 /*
-Package auth implements the basic auth for headers based on "https://github.com/abbot/go-http-auth".
+Package auth provides authentication related filters.
 
-How It Works
+Basic - Check Basic Authentication
 
-The filter accepts two parameters, the first mandatory one is the path to the htpasswd file usually used with Apache or nginx. The second one is the optional realm name that will be displayed in the browser.
-Each incoming request will be validated against the password file, for more information which formats are currently
-supported check "https://github.com/abbot/go-http-auth".
-Assuming you are going to use the MD5 version new entries can be generated like
+The filter accepts two parameters, the first mandatory one is the path to the htpasswd file usually used with
+Apache or nginx. The second one is the optional realm name that will be displayed in the browser. Each incoming
+request will be validated against the password file, for more information which formats are currently supported
+check "https://github.com/abbot/go-http-auth". Assuming that the MD5 version will be used, new entries can be generated like
 
 	htpasswd -nbm myName myPassword
 
-Usage
+Embedding the filter in routes:
+
+	generic: * -> basicAuth("/path/to/htpasswd") -> "https://internal.example.org";
+
+	myRealm: Host("my.example.org")
+	  -> basicAuth("/path/to/htpasswd", "My Website")
+	  -> "https://my-internal.example.org";
+
+Set Basic - Outgoing Authentication Header
```

Comment:
I think the outgoing basic auth was a small feature of skoap, and we are not migrating that in this phase. Please remove this 3 lines from the doc.

Reply:
ok